docs/architecture/adr-020-deterministic-square-construction.md

```diff
@@ -11,7 +11,7 @@ Implemented in <https://github.com/celestiaorg/celestia-app/pull/1690>
 
 ## Context
 
-The current protocol around the construction of an original data square (ODS) is based around a set of constraints that are enforced during consensus through validation (See `ProcessProposal`). Block proposers are at liberty to choose not only what transactions are included and in what order but can effectively decide on the amount of padding (i.e. where each blob is located in the square) and the size of the square. This degree of control leaks needless complexity to users with little upside and allows for adverse behaviour.
+The current protocol around the construction of an original data square (ODS) is based around a set of constraints that are enforced during consensus through validation (See `ProcessProposal`). Block proposers are at liberty to choose not only what transactions are included and in what order but can effectively decide on the amount of padding (i.e. where each blob is located in the square) and the size of the square. This degree of control leaks needless complexity to users with little upside and allows for adverse behavior.
```

Comment:
behaviour is the UK version of behavior

<img width="693" alt="Screenshot 2023-12-16 at 2 58 11 PM" src="https://github.com/celestiaorg/celestia-app/assets/3699047/5e2becd3-81f1-4cb3-89d2-661f0588bf56">